celerybeat.pid

```diff
@@ -0,0 +1 @@
+6
```

Comment:
I don't know why when I run my program it seems this file will be automatically generated. I will remove it.